src/components/search/spellcheck-message.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <Spellcheck v-if="$x.totalResults > 0" #default="{ query }" class="x-message">
     <p>
-      <span v-html="$t('spellcheck.message', { query })" class="x-text" />
+      <span
```

Comment:
we should fix this:

![image](https://user-images.githubusercontent.com/6597815/209327040-a2c9f80c-ae73-4d25-9fb5-022b3bf8416e.png)

Reply:
the button is not align and it doesn't have the right size?

Reply:
I already shared this with the design team. We'll probably have to see how to fix this with the button component.